src/sif/bicyclecost.cc

```diff
@@ -608,11 +608,13 @@ Cost BicycleCost::EdgeCost(const baldr::DirectedEdge* edge,
   }
 
   // Ferries are a special case - they use the ferry speed (stored on the edge)
-  if (edge->use() == Use::kFerry) {
+  if (edge->use() == Use::kFerry || edge->use() == Use::kRailFerry) {
```

Comment:
Actually bike was mostly fine as far as I could make out, this is the only I'd propose